webrtc/RTCSctpTransport-constructor.html

```diff
@@ -0,0 +1,95 @@
+<!doctype html>
+<meta charset="utf-8">
+<title>RTCSctpTransport constructor</title>
```

Comment:
This require quite a bit of refactoring on `interfaces.html` since it involves async initialization. I will put that in a separate PR together with other pending interface tests.